.taskfiles/K0sTasks.yaml

```
@@ -0,0 +1,25 @@
+---
+version: "3"
+
+tasks:
+
+  kubeconfig:
+    desc: Gets k0s cluster kubeconfig
+    cmds:
+      - k0sctl kubeconfig -c k0s-config.yaml > kubeconfig
+    preconditions:
+      - { msg: "k0s-config.yaml not found",      sh: "test -f {{.ROOT_DIR}}/k0s-config.yaml" }
+
+  apply:
+    desc: Apply k0s cluster k0s-config.yaml
+    cmds:
+      - k0sctl apply --config k0s-config.yaml
+      - task: kubeconfig
+    preconditions:
+      - { msg: "k0s-config.yaml not found",      sh: "test -f {{.ROOT_DIR}}/k0s-config.yaml" }
+
+  reset:
+    desc: Resets the k0s cluster
+    cmd: k0sctl reset --config k0s-config.yaml
+    preconditions:
+      - { msg: "k0s-config.yaml not found",      sh: "test -f {{.ROOT_DIR}}/k0s-config.yaml" }
```

```suggestion
    preconditions:
      - { msg: "k0s-config.yaml not found", sh: "test -f {{.ROOT_DIR}}/k0s-config.yaml" }

  apply:
    desc: Apply k0s cluster k0s-config.yaml
    cmds:
      - k0sctl apply --config k0s-config.yaml
      - task: kubeconfig
    preconditions:
      - { msg: "k0s-config.yaml not found", sh: "test -f {{.ROOT_DIR}}/k0s-config.yaml" }

  reset:
    desc: Resets the k0s cluster
    cmd: k0sctl reset --config k0s-config.yaml
    preconditions:
      - { msg: "k0s-config.yaml not found", sh: "test -f {{.ROOT_DIR}}/k0s-config.yaml" }
```

Minor nit on spacing in this playbook, since there's only 1 precondition we don't need to pad out the objects in the map.